AU2/plugins/CorePlugin.py

```diff
@@ -49,6 +49,10 @@ def registered_plugin(plugin_class):
 @registered_plugin
 class CorePlugin(AbstractPlugin):
 
+    PLUGIN_ENABLE_EXPORT: str = "core_plugin_config_update"
+    CONFIG_PARAMETER_EXPORT: str = "core_plugin_edit_config"

```

Comment:
Defining these here makes sense, but now there are inconsistancies in code format between plugins, which I imagine we want to rectify in the long run.
I am also guilty of doing things inconsistantly...

Reply:
Agreed in principle - I don't think I have much of a choice without re-using a string value (and creating inconsistency later potentially)